app/models/work_version.rb

```diff
@@ -28,6 +28,12 @@ class WorkVersion < ApplicationRecord
            dependent: :destroy
   has_many :file_resources,
            through: :file_version_memberships
+  has_many :creations,
```

Comment:
@rschenk these are really aliases, or aliased creators, although future changes could add additional fields and the alias concept might not map anymore. `creations` though, sounds weird, like the work version itself has created something.

Reply:
Yeah good point, creations does sound weird. I'll change that to `aliased_creators` and we can go from there.